ofcourse/test_base/README.md

```diff
@@ -0,0 +1,2 @@
+ofCourse Content Repo
+=====================
```

Comment:
I'm guessing that @decause or @ryansb will want some more content to go into this file?